README.md

```diff
@@ -23,7 +23,51 @@ juju add-unit -n 2 etcd
 
 Its recommended to run an odd number of machines as it has greater redundancy than even number (ie. 4, you can lose 1 before quorum is lost, where as 5, you can 2).
 
+# The charm told me to see the README
 
```

Comment:
So here's what I hope the situation is, after all these changes.

- If you're deploying on Xenial or later, the charm installs the etcd package from the Ubuntu archive, for any supported architecture.

- Otherwise, if you're deploying on amd64, the charm grabs and uses a binary tarball from the CoreOS github account.

- Otherwise - i.e. both old distro and weird architecture - you have to provide the resources yourself.

If that's right, here's what I think the README should say:

You've been directed here because you're deploying this etcd charm onto a pre-Xenial and non-amd64-based Ubuntu platform, and we don't have etcd and etcdctl binaries for that platform.  You will need to obtain such binaries somehow yourself (e.g. by downloading and building from source), then tell Juju about those binaries as detailed below.